### PR TITLE
PAY-2354 Primary-key on calculations table in pay-statement-data shou…

### PR DESCRIPTION
**Pull request by AbidemiAdio**
…ld be combination of calculationId and scheme

### CODE REVIEW

**Comment by marctemp** [comment]
Does this change the data we are expecting to arrive into our service via ServiceBus, i.e. do we need to update our `asyncapi.yaml` to say tht SBI/FRN are string not number?

Also, does this mean we want to convert our `integer`/`bigint` data types for these too?

**Comment by AbidemiAdio** [comment]
> Does this change the data we are expecting to arrive into our service via ServiceBus, i.e. do we need to update our `asyncapi.yaml` to say tht SBI/FRN are string not number?
> 
> Also, does this mean we want to convert our `integer`/`bigint` data types for these too?

We are not changing data-type, neither are we changing content of data we are expecting to arrive into our service, DWH has agreed to fix all the gaps identified.

File: changelog/db.changelog-1.3.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Abidemi Adio" id="1">
+    <tagDatabase tag="v1.3.0" />
+  </changeSet>
+  <changeSet author="Abidemi Adio" id="2">
+    <dropForeignKeyConstraint  baseTableName="fundings" constraintName="fk_calculations_calculationId"/>
```
Comment (marctemp) [inline changelog/db.changelog-1.3.xml:7]: We need to add back our new FK link

**Review by marctemp — CHANGES_REQUESTED** [review]
Update the diagram ERD: https://github.com/DEFRA/ffc-diagrams/blob/main/Payments/ffc-pay-statement-db-schema.drawio

**Comment by AbidemiAdio** [comment]
> Update the diagram ERD: https://github.com/DEFRA/ffc-diagrams/blob/main/Payments/ffc-pay-statement-db-schema.drawio

ERD updated, here is the link to pull request : https://github.com/DEFRA/ffc-diagrams/pull/5

**Review by marctemp — APPROVED** [review]
LGTM

**Pull request merged**